apps/backend/cli_tools/unified_chat_app.py

```
@@ -1018,6 +1033,26 @@ def _show_help(self):
+            except RuntimeError:
+                loop = asyncio.new_event_loop()
+                asyncio.set_event_loop(loop)
+
+            if loop.is_running():
+                import concurrent.futures
+                with concurrent.futures.ThreadPoolExecutor() as executor:
+                    executor.submit(asyncio.run, self.ai_client.shutdown()).result()
+            else:
+                asyncio.run(self.ai_client.shutdown())
+        except Exception:
+            pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)